### PR TITLE
Add Roles and Governance documentation

### PR DESCRIPTION
**Pull request by kingthorin**
To be used for OpenSSF Best Practices Silver.
https://bestpractices.coreinfrastructure.org/en/projects/24?criteria_level=1

"_The project MUST clearly define and document its project governance model (the way it makes decisions, including key roles)._"

"_The project MUST clearly define and publicly document the key roles in the project and their responsibilities, including any tasks those roles must perform. It MUST be clear who has which role(s), though this might not be documented in the same way._"

Signed-off-by: kingthorin <kingthorin@users.noreply.github.com>

### CODE REVIEW

File: site/content/docs/contribute/roles-and-governance.md [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+cascade:
+  EditableContent: true
+  addBreadcrumbs: true
+---
+
+ZAP's governance model is simple.
+
+There are two roles: general contributors, and the [Core Team](/docs/desktop/credits/).
+
+Decisions within the project are made based on majority rules. That is to say, if someone puts forth a new idea or decision to be made, it is made based on the majority of the team agreeing on the direction/outcome.
```
Comment (psiinon) [inline site/content/docs/contribute/roles-and-governance.md:13]: Maybe tweak to say that anyone can work on anything, but it a majority Core Team decision as to whether significant new functionality is adopted (and therefore supported) by the Core Team. All PRs must be approved by 2 members of the Core Team, not including the submitter?

File: site/content/docs/contribute/roles-and-governance.md [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+title: "Roles and Governance"
+type: page
+cascade:
+  EditableContent: true
+  addBreadcrumbs: true
+---
+
+ZAP's governance model is simple.
+
+There are two roles: general contributors, and the [Core Team](/docs/desktop/credits/).
```
Comment (ricekot) [inline site/content/docs/contribute/roles-and-governance.md:11]:
```suggestion
There are two roles: [general contributors](/docs/desktop/credits/#zap-extended-team), and the [Core Team](/docs/team/).
```

File: site/content/docs/contribute/roles-and-governance.md (hunk 1 above)
Comment (kingthorin) [reply site/content/docs/contribute/roles-and-governance.md:13]: Sure

**Review by thc202 — APPROVED** [review]

**Review by psiinon — APPROVED** [review]

**Comment by kingthorin** [comment]
Thanks, I’ll update the silver docs with the new URLs

**Pull request merged**